README.md

```diff
@@ -57,3 +57,10 @@ required for the bot to authenticate itself with Discord. Look for
 You can copy your `config.json` in either: 
 - `config/`
 - `$HOME/marabot/config/`
+
+### Compilation and Running
+
+To compile the bot, run `dotnet build` in the project root.
```

Comment:
You could also run `dotnet build -c Release Marabot.sln` if you wanted to compile a release version of the bot. Not sure if it's worth mentioning?

Reply:
:+1: